stock_storage_type_putaway_strategy/tests/test_storage_type_move.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from .common import TestStorageTypeCommon
+
+
+class TestStorageTypeMove(TestStorageTypeCommon):
+
+    def test_package_level_location_dest_domain(self):
+        pass
```

Comment:
you don't have time to do this tests or they are not commited ?

Reply:
Still WIP